ding/ding.py

```diff
@@ -112,6 +117,7 @@ def main(args=sys.argv[1:]):
         seconds = parse_time(check_input(args))
     except InvalidArguments as e:
         sys.exit(EXIT_MSG.format(e))
+    os.system('cls' if os.name == 'nt' else 'clear') # initial clear
```

Comment:
Do you think it'd be better to have this in the `print_time()` function as the first line of the function? It seems to me that it should be contained in the whole printing functionality.